themes/app/src/scss/app.scss

```diff
@@ -52,6 +52,7 @@
 @import 'base';
 @import 'components/navigation';
 @import 'components/footer';
+@import 'pagetype/homepage';
```

Comment:
I would refactor to this to `components/banner` instead as we can think of a `banner` as more of a component rather than having a dedicated stylesheet just for a page type. 

The idea here is that we want to create styles that are componentised so that we can reuse them in other areas of the site. 

I would then update the classnames below from `homepage__banner-text` to `banner__text` , `banner__images` and so on. Ensure to update the classes on the template level as well :)